lib/time/slots.js

```diff
@@ -69,6 +69,20 @@ function getTime (time) {
 }
 
 /**
+* @method getTimeWithOffset
+* @param offset
+* @return {number}
+*/
+
+function getTimeWithOffset (offset) {
+	var now = new Date().getTime();
+	var time = offset
+		? now - (offset * 1000)
```

Comment:
I know we need to subtract the offset, but usually, offset is added. The API might be confusing. Maybe rather change minus to plus here and use negative offset.